reana_client/cli/__init__.py

```diff
@@ -29,7 +29,55 @@ def __init__(self):
         self.reana_server_url = os.getenv('REANA_SERVER_URL', None)
 
 
-@click.group()
+class ReanaCLI(click.Group):
+    """Reana command line interface."""
+
+    cmd_groups = [ping.configuration_group,
+                  workflow.workflow_group,
+                  files.files_group]
+
+    def __init__(self, name=None, commands=None, **attrs):
+        """Initialize Reana client commands."""
+        click.Group.__init__(self, name, **attrs)
+        for group in ReanaCLI.cmd_groups:
+            for cmd in group.commands.items():
+                self.add_command(cmd=cmd[1], name=cmd[0])
+
+    def list_commands(self, ctx):
+        """Return a list of command names."""
+        return self.commands.keys()
+
+    def get_command(self, ctx, name):
+        """Return command by name."""
+        for cmd in self.commands.items():
+            if name == cmd[0]:
+                return cmd[1]
+
+    def format_commands(self, ctx, formatter):
+        """Overides default click cmd display."""
+        if len(ReanaCLI.cmd_groups):
```

Comment:
We can skip doing `len()` since an empty list would evaluate to `False` as well.